config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
-  get 'bookings/index'
-  get 'bookings/update'
-  get 'bookings/create'
+  get 'chef_profiles/show'
+  get 'chef_profiles/create'
+  get 'courses/index'
+  get 'courses/show'
```

Comment:
We have resources, do we need this? It doesn't look like a complete path either.